openid_connect.js

```diff
@@ -125,7 +145,20 @@ function codeExchange(r) {
 
     // Pass the authorization code to the /_token location so that it can be
     // proxied to the IdP in exchange for a JWT
-    r.subrequest("/_token", "code=" + r.variables.arg_code,
+
+    var internalTokenEndpoint = null;
+    var internalTokenEndpointUri = null;
+
+    if ( r.variables.oidc_pkce_enable == 1 ) {
```

Comment:
Why not pass these as query params instead of needing a completely separate location block? Seems to be adding a lot of extra config lines unnecessarily, but I might have missed something.

Reply:
Totally agree. Let me rewrite this part and use a single location instead of two.